source/accessible-media/playing.html.erb.md

```diff
@@ -59,19 +61,19 @@ following methods:
 Many online media players, on both desktop and mobile platforms, give
 users a wide variety of caption-customization options. For example,
 users can specify caption-font colors, font types and sizes, and
-background colors or transparency levels. Note that in any player on any
-operating system, the settings chosen by the user will override any
-author-specified styling characteristics (see further [comments about
+background colors or transparency levels.
+The settings chosen by the user will override any
+author-specified styling (see further [comments about
 author styling of captions](production-captions.html#a-word-about-styling-captions)).
 
-Below is an image showing caption-styling options in the YouTube desktop
+Below is an image showing caption-styling options in such a
 player.
 
 ![YouTube's caption-styling options: font family, font color, font size,
 and others.](yt_cc_options.png)
 
-Some custom mobile players-- those embedded in Web pages as well as
-stand-alone media players-- also provide styling options from within the
+Some custom mobile players – those embedded in Web pages as well as
+stand-alone media players – also provide styling options from within the
 applications themselves. See the [mobile player used on the CNN Web
 site](http://www.cnn.com/) for one such example.
```

Comment:
pointed to the cnn video page and added instructions for playing videos with captions.